compressor/tests/test_filters.py

```diff
@@ -31,16 +31,20 @@ def blankdict(*args, **kwargs):
 
 @override_settings(COMPRESS_CACHEABLE_PRECOMPILERS=('text/css',))
 class PrecompilerTestCase(TestCase):
+    CHARSET = (
+        settings.FILE_CHARSET if hasattr(settings, 'FILE_CHARSET') else 'utf-8'
+    )
```

Comment:
i think for the tests (here and in that other file), we can indeed just replace every FILE_CHARSET with utf-8. i mean we're reading our own content here, and that should be utf-8...

Reply:
I also replaced them to just `'utf-8'` in the new commit.